components/search-members/search-members.module.scss

```diff
@@ -0,0 +1,12 @@
+.searchContainer {
+  display: flex;
+  align-items: center;
+  padding: 6px;
+}
+.searchBox {
+  float: left;
+  padding: 4px;
+  border: 1px solid rgba(0, 0, 0, 0.397);
+  border-radius: 5px;
+  outline: none;
```

Comment:
What is the purpose of removing outline here...
If it can be reverted it would be beneficial
https://betterprogramming.pub/a11y-never-remove-the-outlines-ee4efc7a9968

Reply:
@harshith-venkatesh Thanks for this blog. I was not aware of the importance of an outline. I will keep the outline.

Reply:
Good one @harshith-venkatesh!
You could also share it in web/css channel on our discord server so more people get to know about it :)